src/Components/LandingPage/LandingPage.tsx

```diff
@@ -62,11 +64,11 @@ const LandingPage = ({ handleCheckboxChange }: LandingPageProps) => {
 
     if (formData.agreeToTermsOfService && formData.is13OrOlder) {
       if (uuid) {
-        navigate(`/${uuid}/step-2`);
+        navigate(`/${uuid}/step-3`);
```

Comment:
Could you use the starting question variable in the steps directory hear?

Reply:
I heard you and have implemented this fantastic suggestion :).